app/Models/Contact/Contact.php

```diff
@@ -982,31 +982,40 @@ public function updateGravatar()
     }
 
     /**
-     * Get the gravatar, if it exits.
+     * Get the first gravatar of all emails, if found.
      *
      * @param  int $size
      * @return string|bool
      */
     public function getGravatar($size)
     {
-        $email = $this->getFirstEmail();
+        $emails = $this->contactFields()
```

Comment:
This is basically the content of `getFirstEmail` method.
This method is now not used anywhere in the app.

Reply:
Indeed. Do you wish to remove it?

Reply:
Yes, if it isn't used...